_includes/module-loading/bundling-required-components.md

```diff
@@ -0,0 +1,11 @@
+The following components are required to bundle {{site.productname}}:
```

Comment:
This section is incorrect, the default icon pack is also a requirement at all times, as other icon packs extend it instead of replace it.

Reply:
A skin is also a mandatory thing, otherwise the editor won't load.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/2035/commits/178d90a9604ea4447bf2872db2856827388f881a